View_User_List.php

```diff
@@ -37,7 +37,38 @@
 			?>
 		</div>
 		<div class="content" style="padding-top:200px;;"><!-- Holds the User table : Username and Password  -->
-			
+			<user1>USERS</user1>
+			<password1>PASSWORD</password1>
+			<h4><section1>
+			<?php
+			//conect.php (tells where to connect servername, password, dbaseName)
+			require "musicdatabase_database_mysqli.php";
+						
+			//create a variable to store sql code for the 'display all users' query
+			$query = ("SELECT * FROM User");
+						
+			//run the query
+			$result = mysqli_query($conn,$query);
+						
+			while ($output = mysqli_fetch_array($result))
+				{
+				echo "<user2>" . $output['User'] . "</user2><br>";
+				//close the while loop
+				}
+			?>
+			</section1></h4>
+			<h4><section2>
+			<?php
+				//run the query
+				$result = mysqli_query($conn,$query);
+						
+				while ($output = mysqli_fetch_array($result))
+					{
+					echo "<user2>" . $output['Password'] . "</user2><br>";
```

Comment:
27/09/2021: This is print out all the value in the Password column and also called it was user2